tools/variant_effect_predictor_105.cwl

```diff
@@ -0,0 +1,190 @@
+      --flag_pick_allele
+      --protein
+      --assembly GRCh38
+      --allele_number
+      --dont_skip
+      --allow_non_variant
+      ${if(inputs.reference) {return "--hgvs --hgvsg --fasta " + inputs.reference.path} else {return ""}}
+      ${if(inputs.cache) {return "--cache --dir_cache ."} else {return ""}}
+      ${if(inputs.species) {return "--species " + inputs.species} else {return ""} }
+      ${if(inputs.merged) {return "--merged"} else {return ""} }
+      ${if(inputs.run_cache_existing) {return "--check_existing"} else {return ""} }
+      ${if(inputs.cadd_indels && inputs.cadd_snvs) {return "--plugin CADD,"+inputs.cadd_snvs.path+","+inputs.cadd_indels.path} else {return ""}}
+      ${if(inputs.run_cache_af) {return "--af_1kg --af_esp --af_gnomad"} else {return ""}}
+      ${if(inputs.dbnsfp) {return "--plugin dbNSFP," + inputs.dbnsfp.path + "," + inputs.dbnsfp_fields} else {return ""}}
+      ${if(inputs.dbscsnv) {return "--plugin dbscSNV,"+inputs.dbscsnv.path} else {return ""}}
+      ${if(inputs.intervar) {return "--custom "+inputs.intervar.path+",Intervar,vcf,exact,0,STATUS"} else {return ""}}
+      | bgzip -c -@ $(inputs.cores) > $(inputs.output_basename).$(inputs.tool_name).vep.vcf.gz &&
+      tabix $(inputs.output_basename).$(inputs.tool_name).vep.vcf.gz
+
+inputs:
+  input_vcf: { type: File, secondaryFiles: [.tbi], doc: "VCF file (with associated index) to be annotated" }
+  ram: {type: 'int?', default: 32, doc: "In GB, may need to increase this value depending on the size/complexity of input"}
+  cores: {type: 'int?', default: 16, doc: "Number of cores to use. May need to increase for really large inputs"}
+  species: {type: 'string?', default: "homo_sapiens", doc: "Refer to the cache dir structure to set this"}
+  buffer_size: {type: 'int?', default: 5000, doc: "Increase or decrease to balance speed and memory usage"}
+  reference: { type: 'File?',  secondaryFiles: [.fai], doc: "Fasta genome assembly with indexes" }
+  cache: { type: 'File?', doc: "tar gzipped cache from ensembl/local converted cache" }
+  merged: { type: 'boolean?', doc: "Set to true if merged cache used", default: true }
+  run_cache_existing: { type: boolean, doc: "Run the check_existing flag for cache" }
+  run_cache_af: { type: boolean, doc: "Run the allele frequency flags for cache" }
+  run_stats: { type: boolean, doc: "Create stats file? Disable for speed", default: false }
+  cadd_indels: { type: 'File?', secondaryFiles: [.tbi], doc: "VEP-formatted plugin file and index containing CADD indel annotations" }
+  cadd_snvs: { type: 'File?', secondaryFiles: [.tbi], doc: "VEP-formatted plugin file and index containing CADD SNV annotations" }
+  dbnsfp: { type: 'File?', secondaryFiles: [.tbi,^.readme.txt], doc: "VEP-formatted plugin file, index, and readme file containing dbNSFP annotations" }
+  dbnsfp_fields: { type: 'string?', doc: "csv string with desired fields to annotate. Use ALL to grab all",
```

Comment:
I understand having this default set at the workflow level but why set it again here?